sorting_algos/radix_sort.py

```diff
@@ -1,28 +1,36 @@
+"""Write a function that accepts an array of positive integers, and returns an array sorted by a radix sort algorithm."""
+
+
 def radix_sort(my_list):
+    """Define a radix sort."""
+    if len(my_list) < 2:
+        return my_list
+        
     RADIX = 10
     maxLength = False
-    tmp, placement = -1, 1
+    tmp = -1
+    placement = 1
 
     while not maxLength:
         maxLength = True
-        # declare and initialize buckets
+        """declare and initialize buckets"""
```

Comment:
Triple quote comments are only for docstrings. Inline comments should always be done with the pound sign, like what you had previously.